api/src/main/java/org/pmiops/workbench/access/AccessModuleServiceImpl.java

```diff
@@ -154,10 +165,7 @@ public boolean isModuleCompliant(DbUser dbUser, AccessModuleName accessModuleNam
 
     // we have an additional check before considering DUCC "complete"
     if (isCompleted && accessModuleName == AccessModuleName.DATA_USER_CODE_OF_CONDUCT) {
-      // protect against NPE when unboxing for comparison
```

Comment:
now stored as a primitive int, so this isn't necessary

Reply:
nit: can it be isCompleted = (isCompleted && accessModuleName == AccessModuleName.DATA_USER_CODE_OF_CONDUCT) && hasUserSignedTheCurrentDucc(dbUser)

Reply:
nvm i think that will not work for others module